static/sass/styles.scss

```diff
@@ -75,6 +72,12 @@ $breakpoint-navigation-threshold: 900px;
   }
 }
 
+.p-navigation>.row {
```

Comment:
This should not be needed. 

I know it's a bit messy with Vanilla only partially supporting using rows inside navigation, but row should keep it's paddings, but we can remove left padding from logo banner, by adding `u-no-padding--left` to `p-navigation__banner`.